src/rf/apps/workers/thumbnail.py

```diff
@@ -47,8 +47,15 @@ def make_thumb(image, thumb_width, thumb_height):
         crop_width = image.width
         crop_height = crop_width / thumb_ratio
 
-    crop_width = int(crop_width)
-    crop_height = int(crop_height)
+    # The band removal factor is a kludge discovered by trial and error
+    # to remove a dark band found on thumbnails of large tiff files.
+    # I'm not sure why this works. I thought that maybe the cropping box
+    # was too big and was going off the edge of the image, but the
+    # crop_width and crop_height never exceed the dimension of the original
+    # image.
+    band_removal_factor = 0.9
+    crop_width = int(crop_width * band_removal_factor)
+    crop_height = int(crop_height * band_removal_factor)
 
     # box = (left, upper, right, lower)
     box = (0, 0, crop_width, crop_height)
```

Comment:
Consider moving the top and left corners by the same offset so this clipping is center aligned.